projects/vdk-heartbeat/src/vdk/internal/heartbeat/job_controller.py

```diff
@@ -23,10 +23,18 @@ class JobController:
 
     def __init__(self, config: Config):
         self.config = config
+        control_api_url = (
```

Comment:
control_api_url_message seems more explicit and clear name for the variable here I think